### PR TITLE
[11092023-mahi]-header,services,testimonial,team,blogs,footer

### PR DESCRIPTION
**Pull request by mahibashardeveloper**

### CODE REVIEW

File: public/css/plugins/all.min.css
Comment (rdhafiz) [inline public/css/plugins/all.min.css:1]: if a plugin has multiple file then make a folder for it. Or rename the file with a suitable plugin name. 
all.min.css meaning nothing that what this css about

File: resources/scss/componets/01_basics.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```
Comment (rdhafiz) [inline resources/scss/componets/01_basics.scss:5]: I think we don't need this cause may be bootstrap already have it. 
Please check and update based on that or let me know if I am wrong

File: resources/scss/componets/01_basics.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+$background-theme-color: #FAF9FF;
+$water-color: rgba(255, 255, 255, 0.1);
+
+// colors variable end
+
+// global start
+
+.background-theme-color{
+    background-color: $background-theme-color !important;
+}
```
Comment (rdhafiz) [inline resources/scss/componets/01_basics.scss]: Move this scss into **background.scss** and remove this from this file please

File: resources/scss/componets/01_basics.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        &:after{
+            left: 0;
+            opacity: 1;
+            width: 100%;
+            color: $theme-color;
+        }
+
+    }
+
+}
```
Comment (rdhafiz) [inline resources/scss/componets/01_basics.scss]: Move this scss into **buttons.scss** and remove this from this file please

File: resources/scss/componets/01_basics.scss [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+        &:after{
+            left: 0;
+            opacity: 1;
+            width: 100%;
+            color: $white;
+        }
+
+    }
+}
```
Comment (rdhafiz) [inline resources/scss/componets/01_basics.scss]: Move this scss into **buttons.scss** and remove this from this file please

File: resources/scss/componets/01_basics.scss [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+.border-theme {
+    border: 1px solid $theme-color;
+}
+
+.border-bottom-theme {
+    border-bottom: 3px solid $theme-color;
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
```
Comment (rdhafiz) [inline resources/scss/componets/01_basics.scss]: Move this scss into **typography.scss** and remove this from this file please

File: resources/scss/componets/02_typography.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
```
Comment (rdhafiz) [inline resources/scss/componets/02_typography.scss:1]: Move this font from here and set it in **resources/views/layouts/components/assets.blade.php**

File: resources/scss/componets/02_typography.scss [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+*{
+    font-family: 'Rubik', sans-serif;
+}
```
Comment (rdhafiz) [inline resources/scss/componets/02_typography.scss:5]: Move this scss into **basics.scss** and remove this from this file please

File: resources/scss/componets/06_header.scss [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+        0%, 80%, 100% {
+            box-shadow: 0 2.5em 0 -1.3em
+        }
+        40% {
+            box-shadow: 0 2.5em 0 0
+        }
+    }
+}
+
+// preloader end
```
Comment (rdhafiz) [inline resources/scss/componets/06_header.scss]: Move this scss into **basics.scss** and remove this from this file please

File: resources/views/layouts/app.blade.php [hunk 9]
```diff
@@ -10,6 +10,8 @@
 
     @include('layouts.components.assets')
 
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"/>
```
Comment (rdhafiz) [inline resources/views/layouts/app.blade.php:14]: Why is this here? Remove it and move all plugin css into **resources/views/layouts/components/assets.blade.php**

File: resources/views/layouts/components/header.blade.php [hunk 10]
```diff
@@ -0,0 +1,43 @@
+<!-- preloader section start -->
+<div id="preloader">
+    <span class="loader"></span>
+</div>
+<!-- preloader section end -->
```
Comment (rdhafiz) [inline resources/views/layouts/components/header.blade.php:5]: Move this to **resources/views/layouts/app.blade.php**

File: resources/views/layouts/components/scripts.blade.php [hunk 11]
```diff
@@ -0,0 +1,7 @@
+<script src="{{asset('js/plugins/jquery.min.js')}}"></script>
+<script src="{{asset('js/plugins/owl.carousel.min.js')}}"></script>
+<script src="{{asset('js/plugins/bootstrap.bundle.min.js')}}"></script>
+
+<script type="application/javascript" src="{{asset('js/main.js')}}"></script>
+<script type="application/javascript" src="{{asset('js/service-carousel.js')}}"></script>
+<script type="application/javascript" src="{{asset('js/testimonial-carousel.js')}}"></script>
```
Comment (rdhafiz) [inline resources/views/layouts/components/scripts.blade.php]: remove this two file and combine the content in one file **homepage.js**

File: public/js/service-carousel.js
[the remark this answers is not in the document]
Comment (mahibashardeveloper) [reply public/js/service-carousel.js:1]: i resolve this carousel source into homepage.js

File: resources/scss/componets/01_basics.scss (hunk 1 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/01_basics.scss:5]: i remove this code now

File: resources/scss/componets/01_basics.scss (hunk 2 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/01_basics.scss]: i remove this scss into background.scss

File: resources/scss/componets/01_basics.scss (hunk 3 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/01_basics.scss]: i move this code into buttons.scss

File: resources/scss/componets/06_header.scss (hunk 8 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/06_header.scss]: i move this preloader code into 01_basic.scss

File: resources/views/layouts/components/scripts.blade.php (hunk 11 above)
Comment (mahibashardeveloper) [reply resources/views/layouts/components/scripts.blade.php]: i move this file in one homepage.js

File: resources/views/layouts/app.blade.php (hunk 9 above)
Comment (mahibashardeveloper) [reply resources/views/layouts/app.blade.php:14]: i face problem for the reason of fontawesome.min.css not support icons. thereby i use this fontawesome from cdnjs. But your order as i remove this link.

File: resources/views/layouts/components/header.blade.php (hunk 10 above)
Comment (mahibashardeveloper) [reply resources/views/layouts/components/header.blade.php:5]: i move this preloader into app.blade.php

File: resources/scss/componets/02_typography.scss (hunk 7 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/02_typography.scss:5]: i move this code into basic.scss

File: resources/scss/componets/01_basics.scss (hunk 4 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/01_basics.scss]: move this code into buttons.scss

File: resources/scss/componets/01_basics.scss (hunk 5 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/01_basics.scss]: move this code into typography,scs

File: resources/scss/componets/02_typography.scss (hunk 6 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/02_typography.scss:1]: remove this code and provide link of google font in assets.blade.php

File: public/js/homepage.js [hunk 12]
```diff
@@ -0,0 +1,29 @@
+// btn-hence working on service button carousel
+
+$('.btn-hence').owlCarousel({
```
Comment (rdhafiz) [inline public/js/homepage.js:3]: why not using a specific name for this slide
do not use css class for js identifier 
give a new meaning full class name for js identifier and use that in js script

File: public/js/homepage.js [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+    responsive: {
+        0: {items: 2},
+        600: {items: 3},
+        1000: {items: 4}
+    }
+});
+
+// testimonial carousel
+
+$('.testimonial').owlCarousel({
```
Comment (rdhafiz) [inline public/js/homepage.js:18]: if this is also a css class then do not use it in js identifier

File: resources/scss/componets/06_header.scss [hunk 14]
```diff
@@ -0,0 +1,151 @@
+// header start
```
Comment (rdhafiz) [inline resources/scss/componets/06_header.scss:1]: remove this comment

File: resources/scss/componets/06_header.scss [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+                color: $theme-color;
+            }
+
+        }
+
+    }
+
+}
+
+// header end
```
Comment (rdhafiz) [inline resources/scss/componets/06_header.scss:151]: remove this comment

File: resources/scss/componets/10_services.scss [hunk 16]
```diff
@@ -0,0 +1,59 @@
+// service start
```
Comment (rdhafiz) [inline resources/scss/componets/10_services.scss:1]: remove this comment

File: resources/scss/componets/10_services.scss [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+        &:last-child {
+            border-left: none;
+            border-right: none;
+        }
+
+    }
+
+}
+
+// service end
```
Comment (rdhafiz) [inline resources/scss/componets/10_services.scss:59]: remove this comment

File: resources/scss/componets/12_testimonials.scss [hunk 18]
```diff
@@ -0,0 +1,97 @@
+// testimonial start
```
Comment (rdhafiz) [inline resources/scss/componets/12_testimonials.scss:1]: remove this comment

File: resources/scss/componets/12_testimonials.scss [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+
+            }
+
+        }
+
+    }
+
+}
+
+// testimonial end
```
Comment (rdhafiz) [inline resources/scss/componets/12_testimonials.scss:97]: remove this comment

File: resources/scss/componets/13_team.scss [hunk 20]
```diff
@@ -0,0 +1,54 @@
+// team start
```
Comment (rdhafiz) [inline resources/scss/componets/13_team.scss:1]: remove this comment

File: resources/scss/componets/13_team.scss [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+                margin: 10px;
+            }
+
+        }
+
+    }
+
+}
+
+// team end
```
Comment (rdhafiz) [inline resources/scss/componets/13_team.scss:54]: remove this comment

File: resources/scss/componets/14_blogs.scss [hunk 22]
```diff
@@ -0,0 +1,45 @@
+// latest news start
```
Comment (rdhafiz) [inline resources/scss/componets/14_blogs.scss:1]: remove this comment

File: resources/scss/componets/14_blogs.scss [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+            padding: 10px 30px;
+            display: inline-block;
+            text-decoration: none;
+            transition: 0.2s all ease-in-out;
+        }
+
+    }
+}
+
+// latest news end
```
Comment (rdhafiz) [inline resources/scss/componets/14_blogs.scss:45]: remove this comment

File: resources/scss/componets/30_footer.scss [hunk 24]
```diff
@@ -0,0 +1,27 @@
+// footer content start
```
Comment (rdhafiz) [inline resources/scss/componets/30_footer.scss:1]: remove this comment

File: resources/scss/componets/30_footer.scss [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+        &:hover {
+            background-color: $theme-color;
+            color: $white;
+        }
+
+    }
+
+}
+
+// footer content end
```
Comment (rdhafiz) [inline resources/scss/componets/30_footer.scss:27]: remove this comment

File: resources/views/layouts/components/assets.blade.php [hunk 26]
```diff
@@ -2,10 +2,12 @@
 <link rel="shortcut icon" href="img/favicon.ico" />
 
 <!-- Google Fonts -->
-
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (rdhafiz) [inline resources/views/layouts/components/assets.blade.php:6]: remove these two lines

**Review by rdhafiz — CHANGES_REQUESTED** [review]

File: public/js/homepage.js (hunk 12 above)
Comment (mahibashardeveloper) [reply public/js/homepage.js]: i provide meaning full class name

File: public/js/homepage.js (hunk 13 above)
Comment (mahibashardeveloper) [reply public/js/homepage.js]: i provide class name which is only identifier

File: resources/scss/componets/06_header.scss (hunk 14 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/06_header.scss]: i remove this comment

File: resources/scss/componets/06_header.scss (hunk 15 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/06_header.scss]: i remove this comment

File: resources/scss/componets/10_services.scss (hunk 16 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/10_services.scss]: i remove this comment

File: resources/scss/componets/14_blogs.scss (hunk 22 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/14_blogs.scss]: i remove this comment

File: resources/scss/componets/14_blogs.scss (hunk 23 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/14_blogs.scss]: i remove this comment

File: resources/scss/componets/30_footer.scss (hunk 24 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/30_footer.scss]: i remove this comment

File: resources/scss/componets/30_footer.scss (hunk 25 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/30_footer.scss]: i remove this comment

File: resources/views/layouts/components/assets.blade.php (hunk 26 above)
Comment (mahibashardeveloper) [reply resources/views/layouts/components/assets.blade.php]: i remove this two link

**Review by rdhafiz — APPROVED** [review]
